python/tvm/relay/prelude.py

```diff
@@ -26,6 +26,42 @@
 from . import op
 
 
+class StaticTensorArrayOps(object):
```

Comment:
How is this related?

Reply:
It's a wrong commit. Just cleaned up the branch.